HCK16_2023_Granada_RH/README.md

```diff
@@ -1,77 +1,106 @@
-# NeuroDataReHack 2
+# NeuroDataReHack 2023
```

Comment:
Did you still want to rename the event Neurodata Rehack? Either way is fine with me.

Reply:
I've been going back and forth on it but I'm leaning towards keeping it as all camel case